changelog.md

```diff
@@ -0,0 +1,132 @@
+-   [Venia directory structure][]([#487][] [#752][])
+
+### App-like experience
+
+-   Pagination feature added to Venia reference storefront for app-like navigation.
+-   Inline checkout implemented in the Venia reference storefront.
+-   Basic account creation and management implemented in Venia.
+-   Loading state implemented to improved the app-like experience.
+
+#### Related documentation updates
+
+-   [Venia Checkout implementation][] ([#673][])
+
+### Improved shopper experience
+
+-   Braintree integration in the Venia reference storefront.
+-   Search with intelligent previews implemented in the Venia reference storefront.
+
+#### Related documentation updates
+
+-   [Braintree integration][]([#793][])
+
+### PWA compliance
+
+-   Web App Manifest file created for the Venia reference storefront.
+    This file provides the metadata for adding the Venia storefront to a mobile home screen.
+-   Service worker optimizations
+
+### Performance enhancements
+
+-   Smart caching implemented to improve service worker catalog caching
+-   Bandwidth usage optimizations achieved using:
+    -   Dynamic image compression/resize
+    -   Advanced minification
+    -   Bundle splitting
+-   Improved app performance on devices using smoother animations and non-blocking renders, which provide faster times on first meaningful paint.
+
+### GraphQL usage
+
+-   GraphQL usage increased to match coverage updates in Magento 2.3.0.
+
+### Magento Cloud compatibility
+
+-   A new Magento module (`Magento_UpwardConnector`) is available to route frontend requests through an UPWARD-PHP server.
+-   Software Integrators(SI) can replace existing Magento cloud storefronts with a new PWA storefront using the UPWARD connector module.
+-   Documentation created for deploying storefront to Magento Cloud
+-   Assistance offered by Magento Enterprise Cloud deployment team
+
+#### Related documentation updates
+
+-   [README][] for the `Magento_UpwardConnector` module, which contains Cloud deployment instructions.
+
+### Other documentation updates
+
+#### New topics
+
+-   [Theme vs Storefront][] ([#509][])
+-   [Install Venia sample data][] ([#474][])
+-   [Peregrine Routing][] ([#453][])
+
+#### Updated topics
+
+-   [Certificate and Valet+ troubleshooting][] ([#637][])
+-   [Tools and Libraries][] ([#621][])
+
+## Known issues
+
+-   After submitting a successful order, users are unable to add items to the cart. ([#916][])
+-   Create account page displayed with null values for email, first and last name when creating an account during a guest checkout. ([#830][])
+-   (Mobile specific) When there are more categories, users cannot scroll to Sign in button. It blocks user from signing to the account. ([#824][])
+
+## Other notable updates
+
+-   **Setup Venia storefront** video developed and available in Magento U
+
+[braintree integration]: https://magento-research.github.io/pwa-studio/venia-pwa-concept/features/braintree/
```

Comment:
I'll try to find a way to autogenerate this as a GitHub query link or something.